pilot/pkg/kube/inject/testdata/inject/policy-check-fail-open.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hello
+spec:
+  replicas: 7
+  template:
+    metadata:
+      annotations:
+        policy.sidecar.istio.io/check: fail-open
```

Comment:
aside: i still think we could use better terminology for this across Istio (allow-on-check-error, deny-on-check-error, etc.).

Reply:
I like allow-on-error, enable, and disable.